com.unity.perception/Runtime/GroundTruth/PerceptionCamera.cs

```diff
@@ -223,27 +188,80 @@ void Update()
 
                 if (!labeler.isInitialized)
                 {
-                    labeler.Init(this, visualizationCanvas);
+                    labeler.Init(this);
                 }
 
                 labeler.InternalOnUpdate();
                 anyVisualizing |= labeler.InternalVisualizationEnabled;
             }
 
+            // TODO - figure this out or talk to Jon about it. Push off to later...
```

Comment:
Could you briefly describe the issue and add the case number to this comment?

Reply:
Done.

Reply:
done